src/process/process.rs

```diff
@@ -265,19 +255,10 @@ impl Process {
 		}
 	}
 
-	/// Preemptively checks if the process is still alive.
 	pub fn terminate(&self) -> ProcessResult<()> {
-		#[cfg(target_family = "unix")]
-		{
-			self.send_signal(Signal::SIGTERM)
-		}
-		#[cfg(not(any(target_family = "unix")))]
-		{
-			todo!()
-		}
+		self.send_signal(Signal::SIGTERM)
```

Comment:
Can you explain this change? Does windows support sigterm?

Reply:
Also we're using unix signals here through the nix crate, so I'm not sure they would work on windows.

Reply:
As explained in https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/signal?view=vs-2019, windows does support signals, but I'm not sure how we can make the `send_signal` method portable, because althrough windows does in fact support signals, I'm not sure how they're sent (except for sending a signal to the current process/thread with `raise`).

Reply:
I think this is good. We'll just have to handle the signals in send_signal